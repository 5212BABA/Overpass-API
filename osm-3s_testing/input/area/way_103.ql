way(103);node(area)(if:id()>=10300 && id()<=10399);out;
