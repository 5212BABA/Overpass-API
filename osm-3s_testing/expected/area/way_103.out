<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base=""/>

  <node id="10301" lat="51.2458880" lon="7.1499776"/>
  <node id="10302" lat="51.2458880" lon="7.1532544"/>
  <node id="10303" lat="51.2491648" lon="7.1532544"/>
  <node id="10304" lat="51.2491648" lon="7.1499776"/>
  <node id="10313" lat="51.2458880" lon="7.1499777"/>
  <node id="10314" lat="51.2458881" lon="7.1499777"/>
  <node id="10315" lat="51.2458881" lon="7.1499776"/>
  <node id="10325" lat="51.2458881" lon="7.1532544"/>
  <node id="10326" lat="51.2458881" lon="7.1532543"/>
  <node id="10327" lat="51.2458880" lon="7.1532543"/>
  <node id="10331" lat="51.2491647" lon="7.1532544"/>
  <node id="10337" lat="51.2491648" lon="7.1532543"/>
  <node id="10338" lat="51.2491647" lon="7.1532543"/>
  <node id="10341" lat="51.2491647" lon="7.1499776"/>
  <node id="10342" lat="51.2491647" lon="7.1499777"/>
  <node id="10343" lat="51.2491648" lon="7.1499777"/>

</osm>
